client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -213,9 +213,13 @@ export const DetailView: FC = () => {
       data as unknown as EncounterSchema,
       updateEncounter
     );
+  const dataStatus = data
+    ? (data as Record<string, JsonData>)['status']
```

Comment:
Not a fan of using this 'as', i would imagine, it's common in program module work, would be great to relocate it further up the chain at some point.